ballista/core/src/execution_plans/distributed_query.rs

```diff
@@ -360,9 +363,22 @@ async fn execute_query(
                 let duration = Duration::from_millis(duration);
 
                 info!("Job {job_id} finished executing in {duration:?} ");
+                let FlightEndpointInfo {
+                    address: flight_proxy_address,
+                } = scheduler
+                    .get_flight_endpoint_info(FlightEndpointInfoParams {})
+                    .await
+                    .map_err(|e| DataFusionError::Execution(format!("{e:?}")))?
```

Comment:
Thinking out loud: Does it need to fail here or it is OK to log it and fallback to no-proxy ?